README.md

```diff
@@ -106,6 +106,12 @@ For more advanced installation options, such as installing to custom locations o
     sudo python setup.py install --register-service
 ```
 
+For Ubuntu, use:
```

Comment:
Rather than making this specific to Ubuntu, can you change it to "For Python 3, use:"

(though that seems redundant to me)